ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.17.yaml

```diff
@@ -79,6 +79,18 @@ releases:
       include_built_images: true
       name: "4.17"
       namespace: ocp
+  multi-initial:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.16"
```

Comment:
this is a y-stream upgrade and we typically test z-stream with same `version` and `relative: 1`

Reply:
yeah I was just copying what was already in there for the standard release and it seems like were using 4.16 > 4.17 for some reason.  Ill change it to do as suggested.